vtm/src/org/oscim/layers/MapEventLayer.java

```diff
@@ -342,7 +342,7 @@ private void onActionMove(MotionEvent e) {
                     mCanTilt = false;
                     mTwoFingersDone = true;
 
-                    mAngle = rad;
+                    mAngle = rad; /* remove to rotate from motion beginning position, not from recognition position */
```

Comment:
What is the meaning of this comment?

Reply:
My intention was to explain, what should be improved:
If you delete the whole line (`mAngle = rad`), you won't notice any big difference. But if you want to rotate the map (e.g. on Android device) you always loose some degrees until the algo recognizes to start rotation mode. So the gesture not really matches the action. But it looks a bit bumpy without animation which should smooth the transition.
Maybe there's a better term for this comment.

Reply:
Maybe it would make sense to write it something like: `smooth the rotation and start from recognized position` or something like this, you probably know better how to describe it. That way it sounds a bit odd I think. Also since you copied the comment multiple times, would it make sense to encapsulate it into a method and giving the method a good speaking name?

Reply:
My english is sometimes not the best :speak_no_evil: 
I'll shorten the comment. I don't wanted to change that much as it should only describe what is done here. And later improve this the right way.

Reply:
Tested those "removals" on device and cannot understand the improvement. :slightly_smiling_face: 

BTW changes in VTM gestures should be optional (via `Parameters`) until we're absolutely sure they work correctly.
e.g. optional `MapEventLayer2` could not be considered production ready since it still has bugs (#280).

Reply:
> Tested those "removals" on device and cannot understand the improvement.

This isn't a improvement yet.
If you change the to `PINCH_ROTATE_THRESHOLD = 1.0f` you'll notice my concern. If removing the line the angles are correct, but the position is a bit weird then.
You always have to rotate more than the human would expect. Even if it's only a small value.

Reply:
If start changing thresholds then strange things would happen, that's for sure.
Thing is if current implementation has issues, using it cannot see any apparent.

Reply:
Can leave it like it is :)
But in my opinion there's a minor lag, that solving would improve user experience. The threshold only overact the problem. It's like the speed improvements, you don't really notice as user, but makes things smoother.

Reply:
Can provide a different implementation to compare before vs after.
And we could always have it as optional if current is still usable.